artemis-core-client/pom.xml

```diff
@@ -65,6 +71,12 @@
          <scope>test</scope>
       </dependency>
       <dependency>
+         <groupId>org.hamcrest</groupId>
```

Comment:
what was bringing that transitively?

Reply:
It's imported by org.apache.activemq.artemis.api.core.TransportConfigurationTest.